index.bs

```diff
@@ -1279,7 +1276,11 @@ in order to reduce resource consumption, notably battery usage.
     : output
     :: None
 
+    1.  Let |sensor| be the [=sensor=] associated with |sensor_instance|.
```

Comment:
Maybe re-arrange steps, first two dedicated to state change, the latter for reading notification? Wdyt?

```
1.  Set |sensor_instance|.{{[[state]]}} to "activated".
1.  [=Fire an event=] named "activate" at |sensor_instance|.
1.  Let |sensor| be the [=sensor=] associated with |sensor_instance|.
1.  If |sensor|’s [=latest reading=]["timestamp"] is not null,
    1.  Queue a task to run [=notify new reading=] with |sensor_instance|
         as an argument.
```